app/src/controllers/housingProject.ts

```diff
@@ -240,8 +240,8 @@ const controller = {
       });
 
       // Create each permit
-      await Promise.all(appliedPermits.map(async (x: Permit) => await permitService.createPermit(x)));
-      await Promise.all(investigatePermits.map(async (x: Permit) => await permitService.createPermit(x)));
+      await Promise.all(appliedPermits.map(async (x: Permit) => await permitService.upsertPermit(x)));
+      await Promise.all(investigatePermits.map(async (x: Permit) => await permitService.upsertPermit(x)));
```

Comment:
The map callback function does not need an async and await - the permitService should be a Promise function already and should be handled by Promise.all's collection.